config/controller.yaml

```diff
@@ -98,6 +98,8 @@ spec:
           value: "false"
         - name: INJECTED_SIDECAR_SUPPORT
           value: "false"
+        - name: EXPERIMENTAL_GENERATE_SLSA_ATTESTATION
```

Comment:
what do you see the process looking like when we want to make this non experimental

Reply:
could we just document it as experimental but keep the env as `GENERATE_SLSA_ATTESTATION` to minimize any tech debt we might take on

Reply:
The only changes needed is to update a single line for parsing command-line/env var flag:
https://github.com/buildpacks-community/kpack/blob/84c91fa7dbecedcf590de63439f02ac510df85cb/cmd/controller/main.go#L96

All the internals use the non-prefixed name
https://github.com/buildpacks-community/kpack/blob/84c91fa7dbecedcf590de63439f02ac510df85cb/pkg/config/config.go#L16
https://github.com/buildpacks-

Reply:
I'm a little hesitant to remove the `EXPERIMENTAL` part since the RFC hasn't been merged and there hasn't been a lot of feedback regarding this feature just yet